### PR TITLE
support contains(string),contains(int) in jsonarray class

### PR DESCRIPTION
**Pull request by cloudbow**
This change is to support searching string in JsonArray since a novice developer may not really know about JsonPrimitves

### CODE REVIEW

File: gson/src/main/java/com/google/gson/JsonArray.java [hunk 1]
```diff
@@ -151,6 +151,28 @@ public JsonElement remove(int index) {
...
+     return contains(new JsonPrimitive(element));     
+  }
+  
+  /**
+   * Overloaded convenience method which searches a Integer in JsonArray and returns true if it exists
+   * Assumes that the JsonArray contains integer elements
+   * @return true if this array contains the specified element.
+   * @param element whose presence in this array is to be tested
+   */
+  public boolean contains(Integer element) {
```
Comment (inder123) [inline gson/src/main/java/com/google/gson/JsonArray.java]: what about other primitive types?

**Comment by inder123** [comment]
marginal value for this PR, adds 2 extra methods.
Sorry, I am inclined to ignore it.

Thank you for the effort nonetheless. Also, appreciate the fact that you wrote a test.

File: gson/src/main/java/com/google/gson/JsonArray.java (hunk 1 above)
Comment (cloudbow) [inline gson/src/main/java/com/google/gson/JsonArray.java]: Json array can contain only integer, string , json objects. Most of us use contains for checking String,Integer. Hence only 2 primitives.

Comment (inder123) [inline gson/src/main/java/com/google/gson/JsonArray.java]: also boolean

**Comment by swankjesse** [comment]
Agreed.

**Pull request closed without merging**